tests/Stampie/Tests/MailerTest.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Stampie\Tests;
 
+
+use GuzzleHttp\Psr7\Response;
```

Comment:
I sometimes use Guzzle/PSR7 just as value objects. But I could rewrite the code to make sure we do not use either.